lib/docker-compose/models/compose_container.rb

```diff
@@ -164,6 +166,14 @@ def prepare_environment(env_entries)
   end
 
   #
+  # Forces the labels structure to use the hash format.
+  #
+  def prepare_labels(labels)
```

Comment:
Instead of assigning a random label I was thinking about using a set of predefined labels the say way as docker-compose (tool) does this: https://github.com/docker/compose/pull/1356/files#diff-84582fcbdd42b004bb763c973ef1e9b5

Using a random label wouldn't give us a solution for maintaining links to containers between restarts of our docker-compose (gem) instance.